docusaurus/docs/iOS/customization/localization.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Translation and i10n
+---
+
+The SDK offers you in-house localization for all strings contained in UI components. For our internal localization, we provide `strings` file for each language that we translate. For advanced plural localization cases (typing indicator showing how many people are typing or how many responses are in thread), we use `stringsdict` with `NSStringPluralRuleType`. 
+
+## The Change of language on the device.
```

Comment:
Change to "Changing the device's language"

Reply:
Changed.